### DISCUSSION TITLE
Positioning of Nodes

### DISCUSSION THREAD

**Discussion by darrenhull** [sgratzl/chartjs-chart-graph#59] [General]
Is it possible to override and set the position of nodes on the canvas? The current layout we are getting is not ideal.

**Comment by sgratzl** [comment]
it depends on the layout that you are choosing
 * the force directed one will automatically position all nodes based on a force simulation
 * the tree / dendogram one will position a tree in a tree shape (hor, vertical, or radial)
 * the generic 'graph' one allow you to specify allow x / y position in the nodes without any automatic layouting
